src/main/scala/com/evolution/scache/CacheMetered.scala

```diff
@@ -162,4 +164,159 @@ object CacheMetered {
       }
     }
   }
+
+  def apply1[F[_]: catshelper.MeasureDuration: Temporal, K, V](
```

Comment:
please ensure `apply` re-uses `apply1`

Reply:
yeah, that makes more sense than copy pasting.. done